src/main/java/Duke/DukeException.java

```diff
@@ -0,0 +1,27 @@
+package Duke;
+
+public class DukeException {
+    void todoException() {
+        System.out.println("OOPS! The description of a todo cannot be empty");
+    }
+    void markException() {
+        System.out.println("Please provide a valid task number to mark");
+    }
+    void unmarkException() {
+        System.out.println("Please provide a valid task number to unmark");
+    }
+    void deadlineException() {
+        System.out.println("Please provide a valid description and deadline.");
+    }
+    void eventException() {
+        System.out.println("Please provide a valid description and time");
+    }
+    void notRecognizedException() {
+        System.out.println("OOPS!!! I'm sorry, but I don't know what that means :-(");
+    }
+    void taskOutOfBounds() {

```

Comment:
Can be renamed to taskOutofBoundsException to be consistent with the other methods here. 
Code Quality Guideline: Name Well